src/librbd/cache/pwl/Types.h

```diff
@@ -240,23 +239,74 @@ struct WriteLogCacheEntry {
   BlockExtent block_extent();
   uint64_t get_offset_bytes();
   uint64_t get_write_bytes();
-  bool is_sync_point() {
-    return sync_point;
+  bool is_entry_valid() const {
+    return flags & write_log_cache_entry_valid;
+  }
+  bool is_sync_point() const {
+    return flags & write_log_cache_entry_sync_point;
+  }
+  bool is_sequenced() const {
+    return flags & write_log_cache_entry_sequenced;
   }
-  bool is_discard() {
-    return discard;
+  bool is_data_stored() const {
```

Comment:
Keep it consistent with the flag name:
```suggestion
  bool has_data() const {
```